### PR TITLE
grep, revbump for rebuild with pcre

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: sys-apps/grep/grep-3.6.recipe [hunk 1]
```diff
@@ -5,50 +5,61 @@ matching lines."
...
 	"
 REQUIRES="
-	haiku
-	lib:libiconv
-	lib:libintl
+	haiku$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libpcre$secondaryArchSuffix
 	"
```
Comment (korli) [inline sys-apps/grep/grep-3.6.recipe:34]: REPLACES is missing for x86_gcc2

Comment (Begasus) [reply sys-apps/grep/grep-3.6.recipe:34]: Need to see how to do that part right :) disabled x86_gcc2 in the previous versions, shouldn't that take care of it?

Comment (korli) [reply sys-apps/grep/grep-3.6.recipe:34]: No, grep_x86 should replace grep when it's already installed on x86_gcc2

Comment (Begasus) [reply sys-apps/grep/grep-3.6.recipe:34]: I think the below section should cover that now (at least it worked here locally)
Should I revert the changes to the prior version (eg enable x86_gcc2 again)?

Comment (korli) [reply sys-apps/grep/grep-3.6.recipe:34]: See the tar recipe for example:
https://github.com/haikuports/haikuports/commit/4b890dcedfcb01de02a18ec8ffa257595ade56ad#diff-14128caa66153dbe9da8ad6b8d6d46569e369c273edc7075a2a483a30e17a1a8R38

Comment (Begasus) [reply sys-apps/grep/grep-3.6.recipe:34]: Didn't think of that ;)

**Review by korli — APPROVED** [review]

**Pull request merged**